src/test/java/lottoAuto/model/StringUtilTest.java

```diff
@@ -0,0 +1,19 @@
+package lottoAuto.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class StringUtilTest {
+
+    @DisplayName("당첨 번호 문자열을 , 기준으로 나눌 수 있다.")
+    @Test
+    public void splitWinningNumberTest() {
+        StringUtil stringUtil = new StringUtil();
+
+        Lotto lotto = stringUtil.splitWinningNumber("1,2,3,4,5,6");
+
+        assertThat(6).isEqualTo(lotto.getLottoNumbers().size());
```

Comment:
아래와 같이 변경해볼 수 있을 것 같아요 :)
```suggestion
        assertThat(lotto.getLottoNumbers()).hasSize(6);
```